code/index.html

```diff
@@ -26,14 +26,23 @@ <h1>Does the person have</h1>
         <optgroup label="hair">
           <option value="brown">brown hair</option>
           <option value="yellow">yellow hair</option>
+          <option value="grey">grey hair</option>
+          <option value="black">black hair</option>
+          <option value="hidden">hidden hair</option>
+          <option value="orange">orange hair</option>
+          <option value="purple">purple hair</option>
+          <option value="white">white hair</option>
           <!-- Add the rest of hair colors as options  -->
         </optgroup>
         <optgroup label="eyes">
-          <option value="hidden">hidden eyes</option>
```

Comment:
The "hidden" eyes is a feature that certain characters have, I would suggest including it :)